pkg/virt-config/virt-config.go

```diff
@@ -513,3 +533,20 @@ func (c *ClusterConfig) GetNetworkBindings() map[string]v1.InterfaceBindingPlugi
 	}
 	return nil
 }
+
+func mergeStringSlices(slice1, slice2 []string) []string {
+	mergedMap := make(map[string]bool)
+	for _, s := range slice1 {
+		mergedMap[s] = true
+	}
+
+	for _, s := range slice2 {
+		mergedMap[s] = true
+	}
+
+	mergedSlice := make([]string, 0, len(mergedMap))
```

Comment:
There is no reason to create the slice with length 0, since it will be filled in the for cycle below.
```suggestion
	mergedSlice := make([]string, len(mergedMap))
```

Reply:
done